component.json

```diff
@@ -59,6 +101,47 @@
         "out": {}
       }
     },
+    "read_action_v2": {
+      "main": "./lib/triggers/read_v2.js",
+      "title": "Read CSV attachment v2",
+      "fields": {
+        "url": {
```

Comment:
Move to `metadata.in`

Reply:
All fields, except "Emit Behavior" moved to metadata